contributor-dashboard-legacy/src/pages/Landing.js

```diff
@@ -380,7 +227,7 @@ function RelativeTimeTillNow(props) {
 	const now = new Date();
 	const then = new Date(props.time);
 	const seconds = (then - now) / 1000;
```

Comment:
This line should probably go below the check of `Number.isNaN(then.getTime())`

Reply:
I suspect we can delete all of `Landing.js`, but fair enough 😄